pkg/template/app/app.go

```diff
@@ -9,6 +11,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/yaml"
 
+	"github.com/giantswarm/k8smetadata/pkg/label"
```

Comment:
Seems to me like imports are all kind of out of order here

Reply:
Does it look correct now?